velox/exec/RowContainer.h

```diff
@@ -302,6 +302,13 @@ class RowContainer {
       char* row,
       int32_t columnIndex);
 
+  /// Stores the 'decoded' into row container at 'column'.
+  void storeVector(
+      const DecodedVector& decoded,
+      vector_size_t size,
```

Comment:
I change the `size` parameter to the type `SelectivityVector`, I think it may be more general. Now this fucntion is only called with the all selected vector.